alerts/zwave_pi4_aeotec.markdown

```diff
@@ -16,3 +16,10 @@ Connect a USB 2.0 hub to the Pi, and the stick into the hub.
 
 Alternatively migrate to using another brand of Z-Wave stick. That would require removing all the devices from the current stick,
 and then including them with the new stick.
+
+## UPDATE
+
+Aeotec has released a New version of the ZStick Gen 5 that does work directly connected to the Raspberry Pi 4 USB2/3 ports.
+Details can be found here: https://community.home-assistant.io/t/new-aeotec-z-stick-now-available-fixes-rpi-4-direct-usb-connect-issues/214186
+which includes a link to purchase the New stick directly. It will take some time for the old sticks to be depleted from retailers.
+Also, Aeotecs backup tool can be used to backup and flash the new stick for easy replacement.
```

Comment:
Small adjustments, furthermore, I'm not sure why we have to link to a community topic here?
Lastly, can we link to `https://aeotec.com/z-wave-usb-stick/` for purchasing instead?

```suggestion
Aeotec has released a new version of the ZStick Gen 5 that does work directly connected to the Raspberry Pi 4 USB2/3 ports. Details can be found here: https://community.home-assistant.io/t/new-aeotec-z-stick-now-available-fixes-rpi-4-direct-usb-connect-issues/214186
which includes a link to purchase the New stick directly. It will take some time for the old sticks to be depleted from retailers.

Also, Aeotecs backup tool can be used to backup and flash the new stick for easy replacement.
```

Reply:
Frank, the community link was added for information only, not required. Also, I included the link provided me in the comm post because, at the time, that was the only way to reliably acquire the NEW stick. It looks like Amazon (in the US listing) on the Aeotec site has been updated, but the third vendor isn't clear. A lot of the old ones are still being sold and folks just need to be aware. However that is best communicated is OK with me. I'll defer to you. Thanks.